README.md

```diff
@@ -146,7 +146,31 @@ require'nvim-treesitter.configs'.setup {
         ["ad"] = "@comment.outer",
         ["am"] = "@call.outer",
         ["im"] = "@call.inner"
-      }
+      },
+      -- swap parameters (keymap -> textobject query)
+      swap_textobject_next = {
+        ["<a-p>"] = "@parameter.inner",
+      },
+      swap_textobject_previous = {
+        ["<a-p>"] = "@parameter.inner",
+      },
+      -- set mappings to go to start/end of adjacent textobjects (keymap -> textobject query)
+      goto_previous_textobject_start = {
+        ["[m"] = "@function.outer",
+        ["[["] = "@class.outer",
+      },
+      goto_previous_textobject_end = {
+        ["[M"] = "@function.outer",
+        ["[]"] = "@class.outer",
+      },
+      goto_next_textobject_start = {
+        ["]m"] = "@function.outer",
+        ["]]"] = "@class.outer",
+      },
+      goto_next_textobject_end = {
+        ["]M"] = "@function.outer",
+        ["]["] = "@class.outer",
+      },
```

Comment:
Why including `textobject` everywhere ? I find that easier if we remove them all.

Reply:
well, swap_previous, swap_next was their previous name that's ok. but how name the goto function. goto_next_start?

Reply:
Yep.

Reply:
We are in the textobject module anyway.

Reply:
but what start?

Reply:
changed it.